templates/weno/admin.html.twig

```diff
@@ -0,0 +1,8 @@
+<head>
```

Comment:
remove this script if it's not being used

Reply:
removed